modules/m123/index.cnxml

```diff
@@ -3,7 +3,7 @@
   <metadata mdml-version="0.5" xmlns:md="http://cnx.rice.edu/mdml">
     <md:title>Introduction</md:title>
     <md:content-id>m123</md:content-id>
-    <md:uuid>11111111-1111-1111-1111-111111111111</md:uuid>
+    <md:uuid>2514fb3b-42a0-4a85-bdf1-9103f88c7974</md:uuid>
```

Comment:
[Here's](https://github.com/openstax/poet/blob/main/server/src/model/page.ts#L57) the regexp for validating UUIDs. 
It requires one of the digits to be 0-5 and another to be `089ab`.

```suggestion
    <md:uuid>11111111-1111-4111-0111-111111111111</md:uuid>
```